modules/lang-painless/src/main/resources/org/elasticsearch/painless/org.elasticsearch.script.ingest.txt

```diff
@@ -40,4 +40,25 @@ class org.elasticsearch.script.Metadata {
 
 class org.elasticsearch.script.IngestScript {
     Metadata metadata()
+    WriteField field(String)
+}
+
+class org.elasticsearch.script.field.WriteField {
+    String getName()
+    boolean isExists()
```

Comment:
Could this simply be named `exists()`?

Reply:
The reason it's called `isExists()` is because painless shortcuts `getX`, `setX` and/or `isX` to `x`, so a script can go `field('foo').exists` rather than `field('foo').exists()`.